docs/UserGuide.md

```diff
@@ -197,7 +195,9 @@ Format: `edit INDEX [n/NAME] [desc/DESCRIPTION] [date/DATE] [s/STATUS] [t/TAG]
 * When editing tags, the existing tags of the task will be removed i.e. adding of tags is not cumulative.
 * You can remove all the task’s tags by typing `t/` without specifying any tags after it.
 * There are 3 Task statuses: `expired`, `uncompleted` and `completed`. However, you can cannot directly modify a 
-  task's status to `expired`
+  task's status to `expired` or from `uncompleted`/`completed` to `expired`.
+* If the edited task exist in the tasklist, an error will be thrown to the user. A task is considered duplicated only if 
+  it's name, description and dateTime is the same.
```

Comment:
dateTime or date?

Reply:
the date consists of time also right ? If like that should be date